### ISSUE TITLE
[Bug] There are two cases of SmartPaymentButton credit card payments: PopUp and Inline.

### ISSUE THREAD

**Issue by ktoyoda39** [paypal/paypal-checkout-components#1479]
### 🐞 Describe the Bug
As shown in the two videos in the attached zip file, there are two cases in SmartPaymentButton, one is when it goes "PopUp" at the time of credit card payment and the other is when it goes "Inline".
I'm using sandbox in Japan, but basically it goes "Inline" most of the time, and sometimes it goes to "PopUp" only once in a while.
Why are there two cases? Doesn't it always behave the same?

[paypal_inline_popup.zip](https://github.com/paypal/paypal-checkout-components/files/5581505/paypal_inline_popup.zip)

### 🔬 Minimal Reproduction
Incognito mode browser - Clear all data for the sandbox domain - SmartPayment button page(Sandbox) like :   
```
<script src="https://www.paypal.com/sdk/js?client-id=sb&currency=JPY&locale=ja_JP&intent=capture&components=buttons,funding-eligibility" data-sdk-integration-source="button-factory"></script>
<script>
  paypal.Buttons({
    style: { shape: 'rect', color: 'gold', layout: 'vertical', label: 'pay', },
    createOrder: function(data, actions) { return actions.order.create({ purchase_units: [{ amount: { value: '1' } }] }); },
    onApprove: function(data, actions) { return actions.order.capture().then(function(details) { alert('Transaction completed by ' + details.payer.name.given_name + '!'); }); }
  }).render('#paypal-button-container');
</script>
```

Then, try to  click "デビッドカードまたはクレジットカード" many times.  

### 😕 Actual Behavior
basically it goes "Inline" most of the time, and sometimes it goes to "PopUp" only once in a while.

### 🤔 Expected Behavior
Always open in "Inline". Or always open in "PopUp".

**Comment by pedrumgolriz** [comment]
With mine, it is doing both. I am developing in electron so I have a native MacOS application. Popups are being blocked, so the popup is empty. I am also getting the inline form which works fine. I'm trying to figure out how to disable the popup entirely

**Comment by jimihay** [comment]
This happens for me as well. It seems like it's almost random. I can refresh the page and then most of the time it will decide to be inline. Also, in the 1 out of 20 times that it decides to open the pop-up, it forces the user to enter their credit card twice. The first screen asks if the user wants to save their card and presents a continue button that says "You will be directed back to the merchant site to complete your order", but when you click continue it reloads what looks like the same page, but this time with a Pay Now button

**Comment by bitstorm** [comment]
This might help you. It's something that always forces the popup mode for credit cards:

https://github.com/paypal/paypal-checkout-components/issues/1521#issuecomment-821815565 

For me it worked.

**Comment by sviluppo** [comment]
In my case the popup is showing only if i click really fast on "black credit/debit cart" before it ends the animation where appears also the text.

It happens also in paypal demo.
https://developer.paypal.com/demo/checkout/#/pattern/responsive

Just refresh the page and click fast on black button before the text "debit or credit card" appear.
